third_party/move/tools/move-mutator/src/lib.rs

```diff
@@ -80,91 +80,85 @@ pub fn run_move_mutator(
     let output_dir = output::setup_output_dir(&mutator_configuration)?;
     let mut report: Report = Report::new();
 
-    for fileid in &env.get_source_file_ids() {
+    for mutant in &mutants {
+        let fileid = &mutant.get_file_id();
```

Comment:
nit: it's more pretty this way
```suggestion
        let file_id = &mutant.get_file_id();
```

Reply:
Done.